lib/github/push.js

```diff
@@ -1,19 +1,25 @@
-const { Project } = require('../models')
-const transformPush = require('../transforms/push')
-const reduceProjectKeys = require('../jira/util/reduce-project-keys')
+const { enqueuePush } = require('../transforms/push')
+const parseSmartCommit = require('../transforms/smart-commit')
 
-module.exports = async (context, jiraClient, util) => {
-  await transformPush(context.payload, context.github, async ({ data: jiraPayload, commands }) => {
-    await jiraClient.devinfo.repository.update(jiraPayload)
-
-    // Don't run Jira commands
-    // await util.runJiraCommands(commands)
+module.exports = async (context, jiraClient) => {
+  // This keeps the context object the same shape, but
+  // filters out any commits that don't have issue keys
+  // so we don't have to process them.
+  context.payload.commits = context.payload.commits.map(commit => {
```

Comment:
In general I would prefer to create a new variable and not mutate the context to avoid any side effects if the context is used somewhere else.

```
const newContext = {
  ...context,
  payload: {
    ...context.payload,
    commits: context.payload.commits.map(...)
  }
}
```